src/Microsoft.Restier.Core/Query/DefaultQueryHandler.cs

```diff
@@ -58,7 +58,7 @@ internal static class DefaultQueryHandler
 
             // execute query
             QueryResult result;
-            var executor = context.GetApiService<IQueryExecutor>();
+            var executor = context.GetApiContextService<IQueryExecutor>();
```

Comment:
Thanks for the reminder. I also meant only those API services **which were previously called hook handlers**. Cannot replace every `GetApiContextService<T>` with `GetApiService<T>`:-)